espressotee/types.go

```diff
@@ -0,0 +1,14 @@
+package espressotee
+
+type ServiceType uint8
+
+const (
+	BatchPoster ServiceType = iota
+	CaffNode
+)
+
+const (
+	Test ServiceType = 2 // Add testing tag at 255 to avoid collisions as this is the least "real" option for a service type.
```

Comment:
I am not sure I understand what we are doing here and why this is needed? Can you expand further on this?